examples/wing-fixture/package.json

```diff
@@ -8,14 +8,23 @@
     "url": "https://wing.cloud",
     "organization": true
   },
+  "scripts": {
+    "compile": "wing compile .",
+    "test": "wing test"
+  },
   "license": "MIT",
   "engines": {
     "node": ">=v18.0.0"
   },
+  "main": "target/wing-fixture.wsim/.wing/preflight.js",
```

Comment:
Adding this allows this library to be used as a symlinked dependency. Not a completely supported usecase, but makes testing consumption from ts much easier

Reply:
TIL